api/src/main/java/ca/bc/gov/educ/api/gradstudent/model/entity/GraduationStudentRecordPaginationEntity.java

```diff
@@ -0,0 +1,88 @@
+package ca.bc.gov.educ.api.gradstudent.model.entity;
+
+import jakarta.persistence.*;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+
+import java.util.Date;
+import java.util.Set;
+import java.util.UUID;
+
+@Data
+@EqualsAndHashCode(callSuper = false)
+@Entity
+@Table(name = "GRADUATION_STUDENT_RECORD")
+public class GraduationStudentRecordPaginationEntity extends BaseEntity {
+
+	@Transient
+    private String pen;
+
+    @Lob
+    @Column(name = "STUDENT_GRAD_DATA", columnDefinition="CLOB")
+    private String studentGradData;
```

Comment:
Not 100% necessary, but if you don't require any information in the student CLOB, you could consider using a DTO projection to make the call a bit lighter/faster.